nlpsandboxclient/client.py

```diff
@@ -387,6 +504,13 @@ def annotate_note(host: str, note: Union[dict, Note],
             annotations = _annotate_person(api_client, text_annotator_req)
         elif tool_type == "nlpsandbox:physical-address-annotator":
             annotations = _annotate_address(api_client, text_annotator_req)
```

Comment:
were we going to update these method names to be more consistent with the `physical-address`, and `person-name` terminology?